### PR TITLE
[DO NOT MERGE] Application tracker storage POC

### PR DESCRIPTION
**Pull request by rachelchenrc**

### CODE REVIEW

File: components/applicationTracker/ApplicationTracker.js [hunk 1]
```diff
@@ -21,21 +27,145 @@ const useStyles = makeStyles(theme => ({
...
   const classes = useStyles();
+  const { userDocRef } = useAuth();
+  const applications = allApplicationLogEntries.map(item => ({
+    id: item.id,
+    document: item.data(),
+  }));
+
+  const handleSave = async () => {
+    const statusEntry = {
+      id: 1, // starting with index 1 when new application added
```
Comment (hannahtwobulls) [inline components/applicationTracker/ApplicationTracker.js:83]: So if I am adding an application, I am setting the `id` on the application's statusEntry to 1? (do we maybe want to have a more descriptive name, like statusId?)

Comment (rachelchenrc) [reply components/applicationTracker/ApplicationTracker.js:83]: yes, when adding an application it's always the first status. I agree we could use a more descriptive name.

Comment (rachelchenrc) [reply components/applicationTracker/ApplicationTracker.js:83]: statusId is good

File: components/applicationTracker/ApplicationTracker.js [hunk 2]
```diff
@@ -21,21 +27,146 @@ const useStyles = makeStyles(theme => ({
...
+      notes: 'i liked the recruiter, jody',
+      status: 'application_submitted',
+      timestamp: firebase.firestore.FieldValue.serverTimestamp(),
```
Comment (rachelchenrc) [inline components/applicationTracker/ApplicationTracker.js:86]: We should use a different way to get timestamp

**Pull request closed without merging**